streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -400,50 +389,50 @@ public void shouldGetExistingStoreLevelSensor() {
             recordingLevel
         );
 
-        verify(metrics);
+        verify(metrics).getSensor("internal.Test worker.task.test-task-1.store.store1.s.sensor1");
```

Comment:
This does not work for me locally. I guess because the name of the thread is `main` for me locally and not `Test worker`. We should avoid using literal thread names in tests. It is not portable.